projects/storefrontlib/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -23,6 +24,11 @@ import { BREAKPOINT } from '../../../layout/config/layout-config';
   selector: 'cx-tab-paragraph-container',
   templateUrl: './tab-paragraph-container.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
```

Comment:
This seems like a lot of complexity to have aria attributes. Can we move and clean up this logic in the template?

Reply:
I tried but adding screen reader properties does not seem to work with ng-templates